packages/scandipwa/src/component/CheckoutOrderSummaryPriceLine/CheckoutOrderSummaryPriceLine.component.js

```diff
@@ -95,18 +95,22 @@ export class CheckoutOrderSummaryPriceLine extends PureComponent {
             return null;
         }
 
-        return (
-            <li block="CheckoutOrderSummary" elem="SummaryItem" mods={ mods }>
-                { this.renderTitle() }
-                <div block="CheckoutOrderSummary" elem="Price">
-                    <strong>
-                        { this.renderPrice() }
-                    </strong>
-                    { this.renderSubPrice() }
-                </div>
-                { children }
-            </li>
-        );
+        if (+price > 0) {
+            return (
+                <li block="CheckoutOrderSummary" elem="SummaryItem" mods={ mods }>
+                    { this.renderTitle() }
+                    <div block="CheckoutOrderSummary" elem="Price">
+                        <strong>
+                            { this.renderPrice() }
+                        </strong>
+                        { this.renderSubPrice() }
+                    </div>
+                    { children }
+                </li>
+            );
+        }
```

Comment:
No nesting like that, just return null if less then 0 same as I suggested to unify  this block with if block on top